### PR TITLE
chore: update loopback-3.x.md

### PR DESCRIPTION
**Pull request by shimks**
First commit adds line breaks, so please review from second commit and onward please :)

## Checklist

- [ ] `npm test` passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the [style guide](http://loopback.io/doc/en/contrib/style-guide.html)
- [ ] Related API Documentation was updated
- [ ] Affected artifact templates in `packages/cli` were updated
- [ ] Affected example projects in `packages/example-*` were updated

### CODE REVIEW

File: docs/site/LoopBack-3.x.md [hunk 1]
```diff
@@ -37,80 +38,90 @@ In the persistence layer, users can contribute the following artifacts:
...
+  4. before/after/afterError hooks at model-level
+  5. before/after/afterError hooks at model method level
 
-LoopBack Next was intentionally designed to allow users to choose their ORM/persistence solution, and our initial version of @loopback/repository is based on juggler 3.x. That makes it possible for users to reuse their existing model definitions, migrating their application incrementally.
+LoopBack 4 was intentionally designed to allow users to choose their own
+ORM/persistence solution, and our initial version of @loopback/repository
+is based on juggler 3.x. That makes it possible for users to reuse their
+existing model definitions, migrating their application incrementally.
+
+LoopBack 4 was intentionally designed to allow users to choose their own
```
Comment (virkt25) [inline docs/site/LoopBack-3.x.md]: Duplicate of above paragrpah.

File: docs/site/LoopBack-3.x.md [hunk 2]
```diff
@@ -37,80 +38,90 @@ In the persistence layer, users can contribute the following artifacts:
...
-| Programming Language  | Built with JavaScript ES5<br>Node.js callback  | TypeScript 2.5.x & JavaScript ES2016/2017<br>Promise & Async/Await         |
-| Core foundation       | Express with LoopBack extensions               | Home-grown IoC container                           |
-| Model Definition      | Models can be defined with JavaScript or json  | Models can be defined with TypeScript/JavaScript/JSON           |
-| Model Persistence     | A model can be attached to a datasource backed by a connector that implements CRUD operations | Repository APIs are introduced to represent persistence related operations. Repository is the binding of model metadata to a datasource |
+| Programming Language  | Built with JavaScript ES5<br>Node.js callback  | TypeScript 2.6.x & JavaScript ES2016/2017<br>Promise & Async/Await |
+| Core foundation       | Express with LoopBack extensions               | Home-grown IoC container                          |
+| Model Definition      | Models can be defined with JavaScript or JSON  | Models can be defined with TypeScript/JavaScript/JSON(TBA) |
```
Comment (virkt25) [inline docs/site/LoopBack-3.x.md]: Why the TBA?

File: docs/site/LoopBack-3.x.md [hunk 3]
```diff
@@ -37,80 +38,90 @@ In the persistence layer, users can contribute the following artifacts:
...
 - Divide the responsibilities of LoopBack models into
-  - Controllers - to handle incoming API requests
-  - Repositories - to provide access to data stores
-  - Models - to define schemas for business objects
-  - Services - to interact with existing REST APIs, SOAP WebServices, and other form of services/microservices
+  - Controllers: handle incoming API requests
+  - Repositories: provide access to datasources
+  - Models: define schemas for business objects
+  - Services: interact with existing REST APIs, SOAP Web Services, and other
+form of services/microservices
```
Comment (virkt25) [inline docs/site/LoopBack-3.x.md]: `form` => `forms`

File: docs/site/LoopBack-3.x.md (hunk 1 above)
Comment (shimks) [reply docs/site/LoopBack-3.x.md]: Oops, forgot to get rid of the paragraph above when I copied it to edit it.

File: docs/site/LoopBack-3.x.md (hunk 2 above)
Comment (shimks) [reply docs/site/LoopBack-3.x.md]: I've put TBA there because I thought we currently don't support creating a `ModelDefinition` through a JSON file.

**Comment by bajtos** [comment]
I don't have any opinion on this change, sorry.

**Comment by shimks** [comment]
@slnode test please

**Comment by shimks** [comment]
@slnode test please

**Comment by shimks** [comment]
@slnode test please

**Pull request merged**